victims/models.py

```diff
@@ -27,6 +29,20 @@ def is_valid(self):
 
         return True
 
+    def to_JSON(self):
+        """
+        Serialize object to json
+        """
```

Comment:
This short docstring could be in one:

```
def to_json(self):
    """Serialize object to json"""
```

But is this docstring really necessary? I feel like the name `to_json` says it all and the docstring is redundant. What do you think about it?